src/parser/jobs/pld/modules/FightOrFlight.tsx

```diff
@@ -229,7 +228,6 @@ export default class FightOrFlight extends Module {
 
 		this.suggestions.add(new TieredSuggestion({
 			icon: ACTIONS.GORING_BLADE.icon,
-			severity: matchClosestLower(SEVERITIES.MISSED_GCD, this.fofErrorResult.goringTooCloseCounter),
```

Comment:
This seems like an unrelated change that shouldn't be in this pull request. (And kind of seems like an unintended change that shouldn't be being removed).

Reply:
This value was already being ignored because it is a TieredSuggestion, unless I misunderstand how JS was working.  (I didn't notice any difference in values for the PLD parses I checked.)  TieredSuggestion calculates its severity (using this matchClosestLower function interally) based on the tiers and value properties.  This had to be removed since Suggestion file was converted to TS and now takes a constructor object instead of an any.

Reply:
Ah, whoops, I misread the surrounding context of the change.